bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissGenericTypical.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+import java.net.DatagramSocket;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.openhab.binding.souliss.internal.network.udp.SoulissCommGate;
+
+/**
+ * This class implements the base Souliss Typical All other Typicals derive from
+ * this class
```

Comment:
can you briefly explain the purpose of the "Typical" - is it a special Souliss Term?